analyses/fusion_filtering/01-fusion-standardization.R

```diff
@@ -0,0 +1,90 @@
+# K. S. Gaonkar 2019
+# Standardizes fusion calls from callers [STARfusion|| Arriba]. The output will have the following columns
+# "Sample"  Unique SampleIDs used in your RNAseq dataset
+# "FusionName" GeneA--GeneB if fusion is intergenic then Gene1A/Gene2A--GeneB  	
+# "Fusion_Type" Type of fusion prediction from caller "inframe","frameshift","other"
+# "Caller" Name of caller used for fusion prediction
+# "JunctionReadCount" synonymous with split reads, provide evidence for the specific breakpoint 
+# "SpanningFragCount" fragment alignment where either mate does not cross the junction
+# "Confidence" Estimated confidence of call from caller, if not available value is "NA"
+# "annots" Annotation from FusionAnnotator
+# 
+# Command line arguments
+# fusionfile : 	Merged fusion calls from [STARfusion|| Arriba]
+#		For STARfusion default columns must be provided
+#		For Arriba along with default columns additional 
+#		columns from Fusion Annotator [fusion_name= GeneA--GeneB; 
+#		annots=Annotation from FusionAnnotator] are appended to the 
+#		right of original arriba data frame 
+#
+# caller :	Caller used for the input merged fusion calls [STARfusion|| Arriba]
+#
+
+library("optparse")
+
+option_list <- list( 
+     make_option(c("-f", "--fusionfile"),type="character",
+        help="Merged fusion calls from [STARfusion || Arriba]"),
+     make_option(c("-c", "--caller"), type="character", 
+        help="Caller type [STARfusion|| Arriba]"),
+     make_option(c("-o","--outputfile"),type="character",
+	help="Standardized fusion calls from [STARfusion || Arriba] (.RDS)")	
+    )
+
+# get command line options, if help option encountered print help and exit,
+
+opt <- parse_args(OptionParser(option_list=option_list))
+
+inputfile<-opt$fusionfile
```

Comment:
Ah yup makes sense, I'll keep that in mind for assignments

Reply:
Not a dumb question at all. Took me a while to figure it out. Overall the answer is _no it does not matter_. Here's how it does make a difference as far as I can tell: 
* If you do this from the user interface, you can add suggestions to batch and then you can commit them in a single commit. This would also give me credit for the commit (which is not something I care about when using this feature, I just find it convenient). The other thing that is sometimes annoying is if you have larger changes you need to make after committing the suggestions, you'll have to pull that before making changes locally. 
* I've found that the downside of the doing everything, including the suggestions, locally is that sometimes I miss things but some folks like using the [resolving conversations feature](https://help.github.com/en/articles/commenting-on-a-pull-request#resolving-conversations) to help with this.

Reply:
Oh yes all credits for edits are yours and @jharenza's :) Thanks! Since I had to make a bunch of edits I think I'll use the resolving button to keep track of editing.